Eshop.WebApi/Controllers/CategoryController.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Eshop.Domain;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Eshop.WebApi.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class CategoryController : Controller
+    {
+        private static List<Category> Categories = new List<Category>
+        {
+            new Category("Computers"),
+            new Category("Mouses")
+        };
+
+        [HttpGet]
+        public List<Category> GetCategory()
+        {
+            return Categories;
+        }
+
+        [HttpGet("{itemType}")]
+        public Category GetCategory(string itemType) 
```

Comment:
Param not changed. You are not careful.

Reply:
fixed - not to categoryName but - just Name